src/pages/cli/migration/identity-claim-changes.mdx

```diff
@@ -0,0 +1,102 @@
+export const meta = {
+  title: `Changing the default identity claim for owner-based auth`,
+  description: ``,
+};
+
+The Amplify CLI is changing the default owner value in GraphQL APIs. Previously, the API stored only the `username` by default. In this next release behind a feature flag, and in an upcoming major CLI release, the GraphQL Transformer will store a user’s unique `sub` and `username`.
```

Comment:
Can the version be called out here so that it is immediately apparent what is impacted?